bigchaindb/models.py

```diff
@@ -165,14 +165,12 @@ class Block(object):
         node_pubkey (str): The public key of the node creating the
             Block.
         timestamp (str): The Unix time a Block was created.
-        voters (:obj:`list` of :obj:`str`): A list of a federation
-            nodes' public keys supposed to vote on the Block.
         signature (str): A cryptographic signature ensuring the
             integrity and validity of the creator of a Block.
     """
 
     def __init__(self, transactions=None, node_pubkey=None, timestamp=None,
```

Comment:
I think `node_pubkey` is also related to the `keypair`?
I am not suggesting to remove it, rather making a note in this PR so that it can be removed in next PR.